### PR TITLE
Added notice for blocks animations plugin

### PR DESCRIPTION
**Pull request by girishpanchal30**
Add notice for blocks animations plugin

Closes https://github.com/Codeinwp/otter-internals/issues/197

@HardeepAsrani I've added a notice for the `Blocks Animations` plugin like `neve-fse` welcome notice.

Once the @harshitarora-in provides the text/image I'll update the text.
Please review this PR and let me know if you need any changes.

### Screenshots 

https://tinyurl.com/22lupjcn

### Checklist before the final review

- [ ] Included E2E or unit tests for the changes in this PR.
- [x] Visual elements are not affected by independent changes.
- [x] It is at least compatible with the [minimum WordPress version](https://wordpress.org/plugins/otter-blocks/).
- [ ] It loads additional script in frontend only if it is required.
- [x] Does not impact the [Core Web Vitals](https://web.dev/vitals/).
- [x] In case of deprecation, old blocks are safely migrated.
- [ ] It is usable in Widgets and FSE.
- [x] Copy/Paste is working if the attributes are modified.
- [x] PR is following [the best practices]()

### CODE REVIEW

**Comment by ineagu** [comment]
@girishpanchal30 my wording was really unfortunate here. The notice would appear in the Blocks Animations plugin and would recommend Otter. The text would stay the same as in Neve FSE

**Comment by girishpanchal30** [comment]
@ineagu Okay, no worries, I'll make changes accordingly.

**Comment by girishpanchal30** [comment]
@ineagu I've moved the notice to `Blocks Animations` plugin and also update the heading text as per the @harshitarora-in mentioned here https://github.com/Codeinwp/otter-internals/issues/197#issuecomment-2246389342.

Ref: https://tinyurl.com/2avsnojt

**Comment by rodica-andronache** [comment]
@girishpanchal30 I've tested with the development build for Otter and Blocks Animation https://vertis.d.pr/i/5qmaCK

- [ ] when I installed Otter I got these errors https://vertis.d.pr/i/6W7FYl
- [ ] if on my test site I had Otter installed and activated ( the current version ), and uploaded Blocks Animation ( From this PR ) I got this error https://vertis.d.pr/i/Jg4Ost
- [ ] on smaller screens, the notice is not responsive https://vertis.d.pr/i/rbKM3h

**Comment by girishpanchal30** [comment]
@rodica-andronache I Fixed the mentioned issue with the latest commit, Please check with the build zip.

**Comment by rodica-andronache** [comment]
@girishpanchal30 I still get this:
- [ ] if on my test site I had Otter installed and activated ( the current version from wp.org ), and uploaded Blocks Animation ( From this PR ) I got this error https://vertis.d.pr/i/Jg4Ost
The other ones are fixed

**Comment by girishpanchal30** [comment]
@rodica-andronache I've tested locally and I can't replicate this issue.
Ref: https://tinyurl.com/24klbf2r

Can you please share details of your test instance here?

File: inc/class-blocks-animation.php [hunk 1]
```diff
@@ -217,6 +222,202 @@ public static function add_frontend_anim_inline_style() {
...
+		// Dismiss after one week from activation.
+		$display_time = get_option( 'otter_animation_welcome_notice_display_time', false );
+		// Save notice display time to auto dismiss after a week.
+		if ( $display_time ) {
+			$display_time = time();
+			update_option( 'otter_animation_welcome_notice_display_time', $display_time );
+		}
+
+		if ( ! empty( $display_time ) && time() - intval( $display_time ) > WEEK_IN_SECONDS ) {
+			update_option( 'otter-animation-dismiss-welcome-notice', 'yes' );
```
Comment (HardeepAsrani) [inline inc/class-blocks-animation.php:397]: To keep with the rest of the plugin, we use snake_case for option names, and bool instead of yes/no.

**Review by HardeepAsrani — CHANGES_REQUESTED** [review]
Looks good, but we should also deactivate the Blocks Animation plugin once the user has activated Otter Blocks to avoid any conflicts.

File: inc/class-blocks-animation.php (hunk 1 above)
Comment (girishpanchal30) [reply inc/class-blocks-animation.php]: Done!

**Comment by girishpanchal30** [comment]
@HardeepAsrani I've skipped the `Blocks Animation` hooks calling instead of deactivating the plugin. Now it's also fixed @rodica-andronache mentioned issue.

**Comment by rodica-andronache** [comment]
@girishpanchal30 now those issues are fixed 👍 
I was only wondering if it's ok that after you click to install & activate the plugin, you need to reload the page to see the installed plugin https://vertis.d.pr/v/K6a7Xo

**Comment by girishpanchal30** [comment]
@rodica-andronache I have hidden the notice (e.g. neve-fse theme). Would you like to force reload the page?

**Comment by rodica-andronache** [comment]
@girishpanchal30 I think so, reload the page, if there isn't other way to see the plugin installed

**Comment by girishpanchal30** [comment]
@rodica-andronache Done! Please check with the latest build zip

**Comment by rodica-andronache** [comment]
@girishpanchal30 it's working well 👍

**Review by HardeepAsrani — APPROVED** [review]

**Pull request merged**